### PR TITLE
Remove react-native cli and add expo bare workflow

### PR DESCRIPTION
**Pull request by albertozaranza**
# What

- Remove react-native cli and add expo bare workflow

# Why

- To improve QA process

# QA

- Install depencies using `yarn`
- Run `yarn start` (if you run with `yarn build:ios` you need run `cd ios && pod install` )

### CODE REVIEW

File: android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,27 +1,31 @@
...
+  <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
+  <queries>
+    <intent>
+      <action android:name="android.intent.action.VIEW"/>
+      <category android:name="android.intent.category.BROWSABLE"/>
+      <data android:scheme="https"/>
+    </intent>
+  </queries>
+  <application android:name=".MainApplication" android:label="@string/app_name" android:icon="@mipmap/ic_launcher" android:roundIcon="@mipmap/ic_launcher_round" android:allowBackup="false" android:theme="@style/AppTheme" android:usesCleartextTraffic="true">
+    <meta-data android:name="expo.modules.updates.EXPO_UPDATE_URL" android:value="YOUR-APP-URL-HERE"/>
```
Comment (julianosbento) [inline android/app/src/main/AndroidManifest.xml:15]: ❓ Is this part right `android:value="YOUR-APP-URL-HERE"` ?

Doesn't need to be a specific value to our app?

File: android/app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -1,27 +1,31 @@
...
+    <meta-data android:name="expo.modules.updates.EXPO_SDK_VERSION" android:value="YOUR-APP-SDK-VERSION-HERE"/>
```
Comment (julianosbento) [inline android/app/src/main/AndroidManifest.xml:16]: ❓ Is this part right `android:value="YOUR-APP-SDK-VERSION-HERE"` ?

Doesn't need to be a specific value to our app?

**Review by julianosbento — APPROVED** [review]

File: android/gradle.properties [hunk 3]
```diff
@@ -21,8 +21,21 @@
...
+# Supported values: expo.jsEngine = "hermes" | "jsc"
+expo.jsEngine=jsc
+
+# Enable GIF support in React Native images (~200 B increase)
+expo.gif.enabled=true
+# Enable webp support in React Native images (~85 KB increase)
+expo.webp.enabled=true
+# Enable animated webp support (~3.4 MB increase)
+# Disabled by default because iOS doesn't support animated webp
+expo.webp.animated=false
```
Comment (lucianomlima) [inline android/gradle.properties:41]: I think we don't use these image types, so I suggest disable all

File: ios/AstroNative/Supporting/Expo.plist [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+        <key>EXUpdatesSDKVersion</key>
+        <string>YOUR-APP-SDK-VERSION-HERE</string>
+        <key>EXUpdatesURL</key>
+        <string>YOUR-APP-URL-HERE</string>
```
Comment (lucianomlima) [inline ios/AstroNative/Supporting/Expo.plist:8]: Needs replace some infos here

**Review by lucianomlima — COMMENTED** [review]
Maybe the files that only get code formatting can be remove from this PR and we can format all codebase in a separate PR

**Comment by albertozaranza** [comment]
This PR was closed because there were problems using custom fonts

**Pull request closed without merging**